### PR TITLE
Update release branch and check CI

### PR DESCRIPTION
**Pull request by dukhovnyi**
Add podspecs for prebuilt xcfraemworks.
MOB-1014

### CODE REVIEW

**Review by igorkravchenko — APPROVED** [review]

File: SocketIO/9.2.0/SocketIO.podspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
```
Comment (gersonnoboa) [inline SocketIO/9.2.0/SocketIO.podspec:1]: Is this SocketIO client the same as the previously named ios-socket-client?

**Review by gersonnoboa — COMMENTED** [review]
Github is complaining that all these files don't have a line at the end of file

File: SocketIO/9.2.0/SocketIO.podspec (hunk 1 above)
Comment (dukhovnyi) [reply SocketIO/9.2.0/SocketIO.podspec:1]: yes, but by some reason, it has been renamed. So, SocketIO and ios-socket-client is the same framework

**Review by gersonnoboa — APPROVED** [review]

**Comment by dukhovnyi** [comment]
!squash

**Comment by dukhovnyi** [comment]
!merge

**Pull request merged**